lib/ttfunk/aggregate.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module TTFunk
+  class Aggregate
+    private
+
+    def coerce(other)
```

Comment:
OK, this is a bit too much. 😅

Reply:
Really? It just lets you combine two instances of an `Aggregate` 😃